app-benchmarks/siege/metadata.xml

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person" proxied="yes">
-		<email>andreacerisara@gmail.com</email>
-		<name>Andrea Cerisara</name>
-	</maintainer>
 	<maintainer type="project" proxied="proxy">
 		<email>proxy-maint@gentoo.org</email>
```

Comment:
you also need to remove `proxy-maint@gentoo.org` and instead add `<!-- maintainer-needed -->`